docs/_src/api/api/reader.md

```diff
@@ -520,7 +500,7 @@ Example:
 **Arguments**:
 
 - `question`: Question string
-- `documents`: List of documents as string type
+- `texts`: List of documents as string type
```

Comment:
As documents have their own type, I would prefer something like "list of texts as string type" here.